scripts/generate-accounts/index.ts

```diff
@@ -7,10 +7,10 @@ import * as dotenv from "dotenv";
 dotenv.config();
 class Accounts {
   public async generate() {
-    if (!fs.existsSync('accounts')) {
-      fs.mkdirSync('accounts');
+    if (!fs.existsSync('./accounts')) {
```

Comment:
Why do you need `./`? Here and everywhere.

Reply:
To mention it as a folder

Reply:
What is wrong with the previous solution?

Reply:
In the ubuntu machine, it is not working.